src/uwtools/config/formats/yaml.py

```diff
@@ -45,6 +53,8 @@ def __repr__(self) -> str:
         The string representation of a YAMLConfig object.
         """
         yaml.add_representer(TaggedString, TaggedString.represent)
+        yaml.add_representer(f90nml.namelist.Namelist, represent_namelist)
```

Comment:
It might look cleaner to
```
from f90nml import Namelist
```
and just
```
yaml.add_representer(Namelist, represent_namelist)
```
here.

But I also see references to both `f90nml.Namelist` and `f90nml.namelist.Namelist` -- is there a difference, or can we use just one? I'm not sure this is accurate but FWIW:
```
>>> f90nml.Namelist is f90nml.namelist.Namelist
True
```
Shorter is better IMO if that's accurate.

Reply:
I had been leaving the full title so as to provide context, but honestly its unnecessary - I'll clean this up

Reply:
There's definitely a place for keeping the explicit namespaces for context and clarity. My gut feeling is that we don't need that much specificity here, but I think you're right to keep it in mind, especially in places where there might be ambiguity.